README.md

```diff
@@ -106,21 +108,35 @@ If you you get a `could not connect to server` error, you could be experiencing
 psql -d foia -c "CREATE USER foia WITH PASSWORD '<<PASSWORD>>';"
 ```
 
+* Change database password in local_settings.py:
+```python
+# Change the password below to the one you use
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'foia',
+        'USER': 'foia',
+        'PASSWORD': 'CHANGETHIS',
+        'HOST': 'localhost',
+        'PORT': '5432',
+    }
+}
+```
+
 * Initialize your database schema:
 
 ```bash
 django-admin.py syncdb
 ```
 
-* Finally, launch the server locally:
+### Elasticsearch setup
 
+* Start the Elasticsearch server
+```bash
+elasticsearch
 ```
-django-admin.py runserver
```

Comment:
Yeah, I think it's in the wrong place under setting up postgres. Maybe I'll move it to a different section.